packages/frontend-2/components/user/profile/EditDialog/Bio.vue

```diff
@@ -16,7 +16,7 @@
             show-label
             show-required
             :rules="[isRequired, isStringOfLength({ maxLength: 512 })]"
-            @change="save"
+            @change="handleNameChange"
```

Comment:
We shouldn't need to track extra events and do manual validation, when we already have vee-validate rules in place. In fact, the rules set up here should already prevent empty names.

It probably doesn't work because we don't run the validation in any way. Introduce `useForm` here and invoke the form's submission on @change